CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - The new `endorsement` configuration entry lets operators set the desired TLS certificate endorsement, either service-endorsed or node-endorsed (self-signed), for each network RPC interface of a node, defaulting to service-endorsed (#2875).
 - A new governance action `request_ledger_chunk` to request the creation of a ledger chunk at the next signature (#3519).
+- A new governance action `request_snapshot` to request the creation of a snapshot at the next signature (#3544).
```

Comment:
Could you also add some brief documentation for these proposals in https://microsoft.github.io/CCF/main/operations/ledger_snapshot.html please?